tests/Feature/FormTest.php

```diff
@@ -62,13 +63,55 @@ public function test_user_can_show_form()
 		$response->assertJson($form->toArray());
 	}
 
+	public function test_show_form_with_time_to_complete()
+	{
+		$user = User::factory()->create();
+		$form = Form::factory()->for($user)->create([
+			'show_time_to_complete' => true
+		]);
+		FormMetrics::factory()->for($form)->create([
+			'total_respondents' => 5,
+			'total_time' => 1000
+		]); 
+
+		$response = $this->actingAs($user)->get("/api/forms/$form->slug");
+
+		$response->assertStatus(200);
+		$response->assertJson($form->toArray());
+
+		$this->assertArrayHasKey('time_to_complete', $response->json());
+		$this->assertNotNull($response->json()['time_to_complete']);
+
+		//verificando se o calculo está correto
+		$this->assertEquals(200, $response->json()['time_to_complete']);
```

Comment:
```suggestion

$total_respondents = 5;
$total_time = 1000;
$expected_result = $total_time / $total_respondents ;
		$this->assertEquals($expected_result, $response->json()['time_to_complete']);
```

Normalmente é uma boa prática evitar "números mágicos", sem contexto.